lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -959,7 +961,18 @@ def view_init(self, elev=None, azim=None):
         else:
             self.azim = azim
 
-    def get_proj(self):
+    def set_proj(self, proj):
+        """
+        Set the type of projection.
```

Comment:
Yes, I think your new setter/getter can be `proj_type`. I am a little leery of using the word "type" in a variable name when one is not referring to a type object, but I think it is ok here, especially if it is documented well.